Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -140,8 +140,26 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
 
     /// Attempts to sends the current forms information to GitHub, on success will redirect the user to the new issue
     @objc func onSend() {
+        if let bodyText = bodyText {
+            self.finishSend()
+        } else {
+        let submitAlert = UIAlertController(title: "Please Provide Description", message: "Are you certain you want to submit this issue without a description?", preferredStyle: UIAlertControllerStyle.alert)
+
+        submitAlert.addAction(UIAlertAction(title: "Submit", style: .default, handler: { (action: UIAlertAction!) in
+            self.finishSend()
+        }))
+
+        submitAlert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: { (action: UIAlertAction!) in
+            submitAlert .dismiss(animated: true, completion: nil)
+        }))
+
+        present(submitAlert, animated: true, completion: nil)
```

Comment:
If I recall correctly from @BasThomas, no need for the completion part here. You can leave it `animated: true)` and it will default completion to `nil`

Reply:
Correct; `completion` has a default expression of `nil` here. :)